src/web/trykkomania/trykkomania.md

```diff
@@ -1,16 +1,18 @@
 ---
-title: Trykkomania
-level: 2
```

Comment:
Et alternativ, er å flytte 'function Ball' osv i en mal som kan forkes, for å lette kopieringen